tools/offline_log_viewer/storage.py

```diff
@@ -136,6 +136,7 @@ class BatchType(Enum):
     cluster_config_cmd = 20
     feature_update = 21
     cluster_bootstrap_cmd = 22
+    version_fence = 23
```

Comment:
A couple more were added in model/record_batch_types.h

Reply:
Fixed